scripts/Deploy-MSBuild.ps1

```diff
@@ -76,7 +76,7 @@ else {
 if ($runtime -eq "Desktop") {
     $targetFramework = "net472"
 } else {
-    $targetFramework = "net8.0"
+    $targetFramework = "net9.0"
```

Comment:
Ah, I forgot we haven't merged #10282! This shouldn't be necessary after that.

Reply:
the Deploy-MSBuild doesn't need to be updated? 
I have update that due to the fact that instructions of this script will be outdated if upgraded to the net9. 
cc: @YuliiaKovalova

Reply:
Closing, please feel free to reopen the comment if it is not the case